sale_order_line_packaging_qty/tests/test_sale_order_line_packaging_qty.py

```diff
@@ -0,0 +1,119 @@
+# Copyright 2020 Camptocamp SA
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl)
+from odoo.exceptions import UserError
+from odoo.tests import TransactionCase
+
+
+class TestSaleOrderLinePackagingQty(TransactionCase):
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        cls.env = cls.env(context=dict(cls.env.context, tracking_disable=True))
+        cls.precision = cls.env["decimal.precision"].precision_get(
+            "Product Unit of Measure"
+        )
+        cls.partner = cls.env.ref("base.res_partner_12")
+        cls.product = cls.env.ref("product.product_product_9")
+        cls.product_no_pckg = cls.env.ref("product.product_product_10")
+        cls.packaging = cls.env["product.packaging"].create(
+            {"name": "Test packaging", "product_id": cls.product.id, "qty": 5.0}
+        )
+        cls.packaging2 = cls.env["product.packaging"].create(
+            {"name": "Test packaging", "product_id": cls.product.id, "qty": 2.2}
+        )
+
+    def test_product_packaging_qty(self):
+        order = self.env["sale.order"].create({"partner_id": self.partner.id})
+        order_line = self.env["sale.order.line"].create(
+            {
+                "order_id": order.id,
+                "product_id": self.product.id,
+                "product_uom": self.product.uom_id.id,
+                "product_uom_qty": 3.0,
+            }
+        )
+        order_line.write({"product_packaging_id": self.packaging.id})
```

Comment:
You can't write to a computed field. For some reason this doesn't throw an error, but the behaviour here (and in other tests) is incorrect.